lutris/gui/lutristray.py

```diff
@@ -50,6 +52,30 @@ def on_quit_application(self, _widget):
         """Callback to quit the program"""
         self.application.do_shutdown()
 
+    @staticmethod
+    def game_oder_key(game):
+        if game["lastplayed"] is None and game["installed_at"] is None:
+            return 0
+        elif game["lastplayed"] is None:
+            return game["installed_at"]
+        elif game["installed_at"] is None:
+            return game["lastplayed"]
+        else:
+            return max(game["lastplayed"], game["installed_at"])
```

Comment:
Thanks for you feedback. Appreciate it.

`game.get` won't work since the key is always present but the value might be set to None. `game[key] or 0` does the trick nicely however. This made the code concise enough to put it in a lambda, which also gets rid of the naming question.